src/js/listeners.js

```diff
@@ -1,23 +1,27 @@
 import { closest } from './polyfill'
-import { Selectors, customProperty, show } from './util'
+import { customProperty, show } from './util'
 
-function clickStepLinearListener (event) {
-  event.preventDefault()
+function buildClickStepLinearListener (options) {
+  return function clickStepLinearListener (event) {
+    event.preventDefault()
+  }
 }
 
-function clickStepNonLinearListener (event) {
-  event.preventDefault()
+function buildClickStepNonLinearListener (options) {
+  return function clickStepNonLinearListener (event) {
```

Comment:
returning an arrow function will be enough here, since you do not use `this` inside